src/components/Sum.jsx

```diff
@@ -1 +1,17 @@
-// export function Sum() {}
+import React from 'react';
+
+export const Sum = ({ a = 0, b = 0 }) => (
+  <p>
+    Sum of
+    {' '}
+    {a}
+    {' '}
+    and
+    {' '}
+    {b}
+    {' '}
+    is
+    {' '}
+    {a + b}
+  </p>
```

Comment:
```suggestion
  <p>Sum of {a} and {b} is {a + b}</p>
```

Reply:
Make it in one line

Reply:
![Issue](https://user-images.githubusercontent.com/106146104/229114390-53451d8f-7d86-445c-86a5-7b8788679b64.png)

linter does not allow me to do so

Reply:
and when I commit a fix code, linter brings my code back